.github/workflows/no-response.yaml

```diff
@@ -0,0 +1,41 @@
+name: No Response
+
+# Both `issue_comment` and `scheduled` event types are required for this Action
+# to work properly.
+on:
+  issue_comment:
+    types: [created]
+  schedule:
+    # Schedule for five minutes after the hour, every 12 hours
+    - cron: '5 */12 * * *'
+
+# By specifying the access of one of the scopes, all of those that are not
+# specified are set to 'none'.
+permissions:
+  issues: write
+
+jobs:
+  noResponse:
+    runs-on: ubuntu-latest
+    if: ${{ github.repository == 'bareos/bareos' }}
+    steps:
+      - uses: godofredoc/no-response@0ce2dc0e63e1c7d2b87752ceed091f6d32c9df09
+        with:
+          token: ${{ github.token }}
+          # Comment to post when closing an Issue for lack of response. Set to `false` to disable
+          closeComment: >
+            Without additional information, we are unfortunately not sure how to
+            resolve this issue. We are therefore reluctantly going to close this
+            bug for now.
+
+            If you find this problem please file a new issue with the same description,
+            what happens, logs and the output of any command used.
+            [knowledge base setdebug level](https://docs.bareos.org/TasksAndConcepts/BareosConsole.html#bcommandsetdebug)
+            All system setups can be slightly different so it's always better to open
+            new issues and reference the related ones.
+
+            Thanks for your contribution.
+          # Number of days of inactivity before an issue is closed for lack of response.
+          daysUntilClose: 45
+          # Label requiring a response.
+          responseRequiredLabel: "waiting for customer response"
```

Comment:
I'd rather use the GitHub default label "question" for this or "need info". 
The end user might not be a customer and IMO long labels don't look nice.

Reply:
Let consolidate around default so `need info` seems the most appropriated .